pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/SchemasResourceBase.java

```diff
@@ -114,6 +114,10 @@ public CompletableFuture<SchemaVersion> deleteSchemaAsync(boolean authoritative,
     }
 
     public CompletableFuture<SchemaVersion> postSchemaAsync(PostSchemaPayload payload, boolean authoritative) {
+        if (SchemaType.BYTES.name().equals(payload.getType())) {
+            return CompletableFuture.failedFuture(new RestException(Response.Status.NOT_ACCEPTABLE,
+                    "Do not upload a BYTES schema, because it's the default schema type"));
+        }
```

Comment:
How does this work if the schema is not currently bytes and someone wants to change it to bytes?

Reply:
@michaeljmarshall I think you can delete all the current schemas of the topic. At this time, it's schema defaults to bytes.

Reply:
BYTES schema doesn't need to upload, when to create producer or consumer with bytes schema, the broker will not check the schema compatibility check